### PR TITLE
Implement klay_getRewards API (KIP82)

### LINKED ISSUE #1657: Implement Rewards API
Implement the Rewards API specified in [KIP-82](https://github.com/yeri-lee/kips/blob/temp/KIPs/kip-82.md).

- https://github.com/klaytn/klaytn/pull/1642
- https://github.com/klaytn/klaytn/pull/1666 : Add `klay_getRewards` RPC

### PR DESCRIPTION
**Pull request by ian0371**
## Proposed changes

This PR implements `klay_getRewards(num)` API. Closes #1657

```
// Request
curl -X POST -H "Content-Type: application/json" --data '{"jsonrpc":"2.0", "method":"klay_getRewards", "params":["0x64fe7e0"],"id":1}' https://api.baobab.klaytn.net:8651
// Response
{
  "jsonrpc":"2.0",
  "id":1,
  "result":{
    "minted": "6400000000000000000",
    "totalFee": "1075975000000000",
    "burntFee": "537987500000000",
    "proposer": "3200268993750000000",
    "stakers": "0",
    "kgf": "2560215195000000000",
    "kir": "640053798750000000"
    "rewards": {
      "0xa86fd667c6a340c53cc5d796ba84dbe1f29cb2f7": "3200268993750000000",
      "0x2bcf9d3e4a846015e7e3152a614c684de16f37c6": "2560215195000000000",
      "0x716f89d9bc333286c79db4ebb05516897c8d208a": "640053798750000000"
    }
  }
}
```

- Describe your changes to communicate to the maintainers why we should accept this pull request.
- If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [ ] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: params/config.go.orig [hunk 1]
```diff
@@ -0,0 +1,577 @@
...
+type ChainConfig struct {
+	ChainID *big.Int `json:"chainId"` // chainId identifies the current chain and is used for replay protection
+
+	// "Compatible" means that it is EVM compatible(the opcode and precompiled contracts are the same as Ethereum EVM).
+	// In other words, not all the hard fork items are included.
+	IstanbulCompatibleBlock  *big.Int `json:"istanbulCompatibleBlock,omitempty"`  // IstanbulCompatibleBlock switch block (nil = no fork, 0 = already on istanbul)
+	LondonCompatibleBlock    *big.Int `json:"londonCompatibleBlock,omitempty"`    // LondonCompatibleBlock switch block (nil = no fork, 0 = already on london)
+	EthTxTypeCompatibleBlock *big.Int `json:"ethTxTypeCompatibleBlock,omitempty"` // EthTxTypeCompatibleBlock switch block (nil = no fork, 0 = already on ethTxType)
+	MagmaCompatibleBlock     *big.Int `json:"magmaCompatibleBlock,omitempty"`     // MagmaCompatible switch block (nil = no fork, 0 already on Magma)
+<<<<<<< HEAD
```
Comment (kjhman21) [inline params/config.go.orig:179]: Do we need to preserve this kinds of conflict statements?

Comment (ian0371) [reply params/config.go.orig]: Wrong file uploaded. sorry for the hassle

**Review by JayChoi1736 — APPROVED** [review]

File: governance/api.go [hunk 2]
```diff
@@ -96,6 +96,28 @@ func (api *GovernanceKlayAPI) GasPriceAt(num *rpc.BlockNumber) (*hexutil.Big, er
...
+	pset, err := api.governance.ParamsAt(num.Uint64())
+	if err != nil {
+		return nil, err
+	}
+	config := pset.ToChainConfig()
+	config.IstanbulCompatibleBlock = api.chain.Config().IstanbulCompatibleBlock
+	config.LondonCompatibleBlock = api.chain.Config().LondonCompatibleBlock
+	config.EthTxTypeCompatibleBlock = api.chain.Config().EthTxTypeCompatibleBlock
+	config.MagmaCompatibleBlock = api.chain.Config().MagmaCompatibleBlock
+	config.KoreCompatibleBlock = api.chain.Config().KoreCompatibleBlock
```
Comment (blukat29) [inline governance/api.go:117]: Let's figure out how to simplify this idiom later.

**Review by blukat29 — APPROVED** [review]

File: governance/api.go (hunk 2 above)
Comment (ian0371) [reply governance/api.go:117]: Let's address this in future PR

**Pull request merged**